pyuvdata/calfits.py

```diff
@@ -257,6 +275,7 @@ def write_calfits(self, filename, run_check=True,
 
         prihdr['CTYPE3'] = ('TIME', 'Time axis.')
         prihdr['CUNIT3'] = ('JD', 'Time in julian date format')
+        prihdr['CRPIX3'] = 1
         prihdr['CRVAL3'] = self.time_array[0]
```

Comment:
Here and elsewhere, is it a requirement of calfits (and uvcal) that the time axis is in order?